taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from .models import Driver, Car
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),

```

Comment:
The `drivers` field in `CarForm` is using `get_user_model().objects.all()`, which retrieves all user instances. This should be filtered to only include `Driver` instances. Consider using `Driver.objects.all()` instead to ensure the queryset is correct.